README.md

```diff
@@ -128,540 +125,24 @@ eval $cmd
-
-LoRA with rank 128, batch size 1, gradient checkpointing, optimizer adamw, `49x512x768` resolutions, **without precomputation**:
-
-```
-Training configuration: {
-    "trainable parameters": 163577856,
-    "total samples": 69,
-    "train epochs": 1,
-    "train steps": 10,
-    "batches per device": 1,
-    "total batches observed per epoch": 69,
-    "train batch size": 1,
-    "gradient accumulation steps": 1
-}
-```
-
-| stage                   | memory_allocated | max_memory_reserved |
-|:-----------------------:|:----------------:|:-------------------:|
-| before training start   | 38.889           | 39.020              |
-| before validation start | 39.747           | 56.266              |
-| after validation end    | 39.748           | 58.385              |
-| after epoch 1           | 39.748           | 40.910              |
-| after training end      | 25.288           | 40.910              |
-
-Note: requires about `59` GB of VRAM when validation is performed.
-
-LoRA with rank 128, batch size 1, gradient checkpointing, optimizer adamw, `49x512x768` resolutions, **with precomputation**:
-
-```
-Training configuration: {
-    "trainable parameters": 163577856,
-    "total samples": 1,
-    "train epochs": 10,
-    "train steps": 10,
-    "batches per device": 1,
-    "total batches observed per epoch": 1,
-    "train batch size": 1,
-    "gradient accumulation steps": 1
-}
-```
-
-| stage                         | memory_allocated | max_memory_reserved |
-|:-----------------------------:|:----------------:|:-------------------:|
-| after precomputing conditions | 14.232           | 14.461              |
-| after precomputing latents    | 14.717           | 17.244              |
-| before training start         | 24.195           | 26.039              |
-| after epoch 1                 | 24.83            | 42.387              |
-| before validation start       | 24.842           | 42.387              |
-| after validation end          | 39.558           | 46.947              |
-| after training end            | 24.842           | 41.039              |
-
-Note: requires about `47` GB of VRAM with validation. If validation is not performed, the memory usage is reduced to about `42` GB.
-
 </details>
 
-If you would like to use a custom dataset, refer to the dataset preparation guide [here](./assets/dataset.md).
-
-> [!NOTE]
-> To lower memory requirements:
-> - Use a DeepSpeed config to launch training (refer to [`accelerate_configs/deepspeed.yaml`](./accelerate_configs/deepspeed.yaml) as an example).
-> - Pass `--precompute_conditions` when launching training.
-> - Pass `--gradient_checkpointing` when launching training.
-> - Pass `--use_8bit_bnb` when launching training. Note that this is only applicable to Adam and AdamW optimizers.
-> - Do not perform validation/testing. This saves a significant amount of memory, which can be used to focus solely on training if you're on smaller VRAM GPUs.
-
-## Memory requirements
```

Comment:
We don't need as have it here already:
https://github.com/a-r-r-o-w/finetrainers/tree/main/training